src/quo2/components/buttons/predictive_keyboard/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.buttons.predictive-keyboard.view
+  (:require [react-native.core :as rn]
+            [quo2.components.buttons.predictive-keyboard.style :as style]
+            [quo2.components.info.info-message :as info-message]
+            [react-native.linear-gradient :as linear-gradient]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.buttons.button :as button]))
+
+(def gradients
+  {:light [(colors/alpha colors/neutral-40 0.05) (colors/alpha colors/neutral-40 0)]
+   :dark  [(colors/alpha colors/neutral-80 0.7) (colors/alpha colors/neutral-80 0)]
+   :blur  [colors/white-opa-5 colors/white-opa-0]})
+
+(defn- render-word
+  [word {:keys [blur? on-press]}]
+  [button/button
+   (merge {:type :blur-bg :size 32 :on-press #(on-press word)} (when blur? {:override-theme :dark}))
+   word])
+
+(defn- separator
+  []
+  [rn/view {:width 8}])
+
+(defn view
+  "Options
+   - `type` `:words`/`:error`/`:info`/`:empty`.
+   - `blur?` Boolean to enable blur background support.
+   - `text`  error/info text.
+   - `words` List of words to display in the keyboard.
+   - `on-press` Callback called when a word is pressed `(fn [word])`."
+  [{:keys [type blur? text words on-press]}]
+  [linear-gradient/linear-gradient
+   {:style               (style/wrapper type)
+    :accessibility-label :predictive-keyboard
+    :colors              (if blur?
+                           (gradients :blur)
+                           (colors/theme-colors (gradients :light) (gradients :dark)))}
+
+   (condp = type
+     :words
+     [rn/flat-list
+      {:data                              words
+       :content-container-style           style/word-list
+       :render-fn                         (fn [word] [render-word word
+                                                      {:blur? blur? :on-press on-press}])
+       :shows-horizontal-scroll-indicator false
+       :separator                         [separator]
+       :horizontal                        true
+       :key-fn                            str}]
+
+     :error
+     [info-message/info-message
+      {:icon :i/info
+       :size :default
+       :type :error} text]
+
+     :info
+     [info-message/info-message
+      (merge {:icon :i/info
+              :size :default
+              :type (if (= type :error) :error :default)}
+             (when blur?
+               {:text-color colors/white-opa-70
+                :icon-color colors/white-opa-70})) text]
```

Comment:
Please, align multiline args vertically, as is explained in the style guide https://guide.clojure.style/#vertically-align-fn-args. zprint won't do this for us. In this line in particular, it's quite difficult to grasp that `text` is the argument of `info-message`.